examples/Analytic_300hPa_Trough.py

```diff
@@ -0,0 +1,179 @@
+"""
+Analytic 300-hPa Trough
+=======================
+
+"""
+
+import matplotlib.pyplot as plt
+import metpy.calc as mpcalc
+from metpy.units import units
+import numpy as np
+
+
+######################################################################
+# Below are three definitions to create an analytic 300-hPa trough roughly
+# based on the Sanders Analytic Model with modified coefficients to create
+# different style waves.
+#
+
+def single_300hPa_trough(parameter='hght'):
+    """ Single trough with heights and Temperatures based on Sanders Analytic Model
+    """
+    X = np.linspace(.25, .75, 101)
+    Y = np.linspace(.25, .75, 101)
+
+    x, y = np.meshgrid(X, Y)
+
+    p = 4
+    q = 2
+
+    if parameter == 'hght':
+        return (9240 + 100 * np.cos(p * x * np.pi) * np.cos(q * y * np.pi)
+                + 200 * np.cos(y * np.pi) + 300 * y * np.cos(x * np.pi + np.pi / 2))
+    elif parameter == 'temp':
+        return (-50 + 20 * np.cos(p * x * np.pi) * np.cos(q * y * np.pi) / 10
```

Comment:
Multiplying by 20 then dividing by 10 seems like it could be collapsed. Is there some reason not to (like matching the formulation in a reference)?